app/components/emoji_keyboard.js

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {
+    View,
+    Text,
+    StyleSheet,
+    FlatList,
+    TouchableOpacity
+} from 'react-native';
+import {KeyboardRegistry} from 'react-native-keyboard-input';
+import {store} from 'app/mattermost';
+import {CategoryNames, EmojiIndicesByCategory, Emojis} from 'app/utils/emojis';
+import Emoji from 'app/components/emoji/index';
```

Comment:
make this `import Emoji from 'app/components/emoji';` you don't need to specify the `index` in the path